src/xyArray/__tests__/xyArrayAlign.test.ts

```diff
@@ -1,4 +1,4 @@
-import { xyArrayAlign } from '../xyArrayAlign.js';
+import { xyArrayAlign } from '../xyArrayAlign';
```

Comment:
Did you remove the `.js` because of an error? If yes, what was it?

Reply:
Because the file is no longer .js

Reply:
Unable to resolve path to module '../xyArrayAlign.js'.